src/Static.js

```diff
@@ -0,0 +1,121 @@
+
+import React from 'react';
+import PropTypes from "prop-types";
+import './Static.css'
+import Container from 'react-bootstrap/Container';
+import Row from 'react-bootstrap/Row';
+import { warningIcon, checkMarkIcon } from './encodedImages';
+import OverlayTrigger from 'react-bootstrap/OverlayTrigger';
+import Tooltip from 'react-bootstrap/Tooltip';
+/*global chrome*/
+
+const importStatusEnum = {
+    none: 1,
+    error: 2,
+    success: 3
+}
+
+let checked = false;
+
+class Static extends React.Component {
+
+    constructor() {
+        super();
+
+        this.state = {
+            importStatus: importStatusEnum.none,
+            importSettingsTitle: "Import settings",
+            errorDescription: "Something went wrong when importing your custom setting file. Please try again or proceed with default settings."
```

Comment:
Similar question, should we also import these from somewhere?

Reply:
It's also part of setting the state of the importation. 
```
window.setImportStatus({
    status: 2,
    importSettingsTitle: 'Failed to Import Settings',
    errorDescription: 'Something went wrong when importing your custom setting file. Please try again or proceed with default settings.'
})
```